### PR TITLE
Adding Sentry support to logger package

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

Adding Sentry support to the logger package. With this change, the error (or worst) log messages can be sent directly to the configured Sentry project.

## Type of change

- New feature (non-breaking change which adds functionality)
- Documentation update

## Testing steps

Regular CI

### CODE REVIEW

**Review by dpensi — APPROVED** [review]

File: docs/packages/logger/configuration.html [hunk 1]
```diff
@@ -173,6 +173,15 @@
 (configuration for CloudWatch is in CloudWatchConfiguration)</p>
 </td>
 	<td class="code"><pre><code>	<div class="ident">LoggingToCloudWatchEnabled</div> <div class="ident">bool</div> <div class="literal">`mapstructure:&#34;logging_to_cloud_watch_enabled&#34; toml:&#34;logging_to_cloud_watch_enabled&#34;`</div><div class="operator"></div>
+
+</code></pre></td>
+      </tr>
+      
+      <tr class="section">
+	<td class="doc"><p>LoggingToSentryEnable enables logging to Sentry
+(configuretion for Sentry is in SentryLoggingConfiguration)</p>
```
Comment (epapbak) [inline docs/packages/logger/configuration.html:182]: configuration

File: logger/configuration.go [hunk 2]
```diff
@@ -34,6 +34,10 @@ type LoggingConfiguration struct {
 	// LoggingToCloudWatchEnabled enables logging to CloudWatch
 	// (configuration for CloudWatch is in CloudWatchConfiguration)
 	LoggingToCloudWatchEnabled bool `mapstructure:"logging_to_cloud_watch_enabled" toml:"logging_to_cloud_watch_enabled"`
+
+	// LoggingToSentryEnable enables logging to Sentry
```
Comment (epapbak) [inline logger/configuration.go:38]: LoggingToSentryEnabled

**Review by epapbak — APPROVED** [review]
Some small comments on comments

File: docs/packages/logger/configuration.html (hunk 1 above)
Comment (joselsegura) [reply docs/packages/logger/configuration.html:182]: Good catch

File: logger/configuration.go (hunk 2 above)
Comment (joselsegura) [reply logger/configuration.go:38]: Cool, tyvm

**Review by Bee-lee — APPROVED** [review]
lgtm

**Pull request merged**